app/views/devise/registrations/edit.html.erb

```diff
@@ -29,6 +29,12 @@
       <%= raw t('devise.registrations.google_auth1') %>
     </div>
   <% end %>
+  
+  <% if current_user.provider == 'facebook' %>
```

Comment:
It would be great to add an acceptance test here for this and Google as auth well.